### ISSUE TITLE
Blank Node Scoping

### ISSUE THREAD

**Issue by jeswr** [comunica/comunica#1123]
#### Issue type:

- :heavy_plus_sign: Feature request 

This is a feature request for blank node scoping in documents - for details see discussion in https://github.com/comunica/comunica/issues/911

____

**Comment by rubensworks** [comment]
Could you summarize the issue in the main post?
Might be a bit confusing for people interested in this issue having to read through all of #911 first.

**Comment by rubensworks** [comment]
@jeswr I'm still not sure what the goal of this issue is.
As to not to pollute the issue tracker with unclear tasks, I'll close this one but feel free to re-open it after updating the description of the issue.
